src/test/java/com/github/dockerjava/cmd/ListContainersCmdIT.java

```diff
@@ -28,33 +36,56 @@
 
 public class ListContainersCmdIT extends CmdIT {
     private static final Logger LOG = LoggerFactory.getLogger(ListContainersCmdIT.class);
+    private static final String DEFAULT_IMAGE = "busybox";
 
-    @Ignore("can't work in parallel and second test seems cover this")
-    @Test
-    public void testListContainers() throws Exception {
+    private static Map<String, String> testLabel;
```

Comment:
@fbuecklers static probably is what causing test failure?

Reply:
Yes totally true... I haven't changed that after I switched to instance based test uuid
Thanks!